blog/2024-03-05-nushell_0_91_0.md

```diff
@@ -25,15 +27,8 @@ The optional dataframe functionality is available by `cargo install nu --feature
 As part of this release, we also publish a set of optional plugins you can install and use with Nu. To install, use `cargo install nu_plugin_<plugin name>`.
 
 # Table of content
-- [*Themes of this release / New features*](#themes-of-this-release-new-features-toc)
-    - [*Hall of fame*](#hall-of-fame-toc)
-        - [*Bug fixes*](#bug-fixes-toc)
-        - [*Enhancing the documentation*](#enhancing-the-documentation-toc)
-    - [*Our set of commands is evolving*](#our-set-of-commands-is-evolving-toc)
-        - [*New commands*](#new-commands-toc)
-        - [*Changes to existing commands*](#changes-to-existing-commands-toc)
-        - [*Deprecated commands*](#deprecated-commands-toc)
-        - [*Removed commands*](#removed-commands-toc)
+
+- [_Themes of this release / New features_](#themes-of-this-release-new-features-toc) - [_Plugin protocol overhaul_](#plugin-protocol-overhaul) - [_Hall of fame_](#hall-of-fame-toc) - [_Bug fixes_](#bug-fixes-toc) - [_Enhancing the documentation_](#enhancing-the-documentation-toc) - [_Our set of commands is evolving_](#our-set-of-commands-is-evolving-toc) - [_New commands_](#new-commands-toc) - [_`tee`_](#tee) - [_`interleave`_](#interleave) - [_Changes to existing commands_](#changes-to-existing-commands-toc) - [_`zip` supports closures_](#zip-supports-closures) - [_Deprecated commands_](#deprecated-commands-toc) - [_Removed commands_](#removed-commands-toc)
```

Comment:
looks like line endings got eaten or something?

Reply:
oh weird, the autoformatter that runs on `git commit` (lefthook?) must have done that. very odd